Transactions.md

```diff
@@ -30,9 +30,11 @@ The funding inputs and change output script public keys are negotiated in the of
 
 ## Funding Transaction Input and Output Ordering
 
-The Offerer/Initiator's inputs come first and in the order that they were listed in the Offer message, the Accepter's inputs come after this in the order they are listed in the Accept message. The funding output comes first, followed by the Offerer's change output if applicable, and then the Accepter's change output if applicable.
+The inputs are sorted by each `funding_input`'s `input_serial_id` in ascending order.
+The outputs are sorted by `output_serial_id` in ascending order with the funding output having an `output_serial_id` of `0x8000000000000000`
 
-Note that this will likely change in the future.
+The funding output has a `output_serial_id` of `0x8000000000000000` since it is the midpoint of an uint64.
+This is chosen to give the funding output the most random index among the outputs.
```

Comment:
I'm a bit dubious about that. Tbh this const makes thing more complicated, and I have trouble understanding how you can make something more random by using a constant (and so this rational didn't do a good enough job at convincing me :) )

Reply:
Updated, lmk what you think